src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -160,6 +160,39 @@ properties:
     type: number
     minimum: 0
 
+  margin_distance:
+    $ref:
+      - "_terms.yaml#/margin_distance/common"
+    type: number
+    minimum: 0
+
+  margin_status:
+    $ref:
+      - "_terms.yaml#/margin_status/common"
+    enum:
+      - Indeterminate
+      - Involved
+      - Uninvolved
+      - Unknown
+      - Not Reported
+
+  margins_involved_site:
+    $ref:
+      - "_terms.yaml#/margins_involved_site/common"
+    enum:
+      - Cervix uteri
```

Comment:
should "Cervix uteri" be capitalized? now it's like in the task

Reply:
updated 👍